docs/archive-data.md

```diff
@@ -4,99 +4,75 @@ title: How to archive data
+Temporal supports several providers out of the box.
 
-### Archival configuration
+- **Local file system**: The [filestore archiver](https://github.com/temporalio/temporal/tree/master/common/archiver/filestore) is used to archive data in the file system of whatever host the Temporal server is running on. This provider is used mainly for local installations and testing and should not be relied on for production environments.
+- **Google Cloud**: The [gcloud archiver](https://github.com/temporalio/temporal/tree/master/common/archiver/gcloud) is used to connect and archive data with [Google Cloud](https://cloud.google.com/storage).
+- **S3**: The [s3store archiver](https://github.com/temporalio/temporal/tree/master/common/archiver/s3store) is used to connect and archive data with [S3](https://aws.amazon.com/s3). 
+- **Custom**: If you want to use a provider that is not currently supported, you can [create your own archiver](#custom-archiver) to support it.
 
-First familiarize yourself with the configuration in [`config/development.yaml`](https://github.com/temporalio/temporal/blob/master/config/development.yaml#L81).
+When you choose a provider, make sure to save the exact storage location's URI, as it is passed as a parameter when you [create a Namespace](#namespace-creation).
 
-| Config |  Description |
-|--------|--------------|
-| `archival.history.state`                       | Acceptable values are `"enabled"`,`"disabled"`. This must be set to `"enabled"` to use the Archival feature with any Namespace in the cluster. |
-| `archival.history.enableRead`                  | Acceptable values are `true`, `false`. This must be set to `true` to read from the archived Event History. |
-| `archival.history.provider`                    | Sub `provider` configs are `filestore`, `gstorage`, `s3`, or `your_custom_provider`. The default config specifies `filestore`. |
-| `archival.history.provider.filestore.fileMode` | This specifies the file permissions of the archived files. We recommend using the default value of `"0666"` to avoid read/write issues. |
-| `archival.history.provider.filestore.dirMode`  |  This specifies the directory permissions of the archive directory. we recommend using the default value of `"0766"` to avoid read/write issues. |
-| `namespaceDefaults.archival.history.state`     | Acceptable values are `"enabled"`, `"disabled"`. This sets the default state of the Archival feature whenever a new Namespace is created without specifying the Archival state. |
-| `namespaceDefaults.archival.history.URI`       | The value must be a URI of the file store location and match a schema that correlates to a provider. |
+### Configuration
 
-Consider the following example:
+Archival configuration is defined in the [`config/development.yaml`](https://github.com/temporalio/temporal/blob/master/config/development.yaml#L81) file.
+
+In the following example configuration we see that:
```

Comment:
I think it would be cool if you inlined the lines you are referring to in this list. Right now I have to manually match it myself.